lib/elements/dom-module.html

```diff
@@ -124,16 +130,14 @@
     register(id) {
       id = id || this.id;
       if (id) {
-        if (Polymer.strictTemplatePolicy && findModule(id)) {
-          modules[id] = lcModules[id.toLowerCase()] = null;
-          throw new Error(`strictTemplatePolicy: dom-module ${id} registered twice`);
+        // Under strictTemplatePolicy, reject and null out any re-registered
+        // dom-module since it is ambiguous whether first-in or last-in is trusted 
+        if (Polymer.strictTemplatePolicy && findModule(id) !== undefined) {
```

Comment:
Add a test that tries to register the same module 3x.

Reply:
Done.